webapp/users.py

```diff
@@ -18,33 +34,31 @@ def __init__(self, name, link='/remote'):
         self.name = name
         self.link = link
 
-class User(UserMixin):
-    """A class for instantiating saved user accounts.
-        There should be 1 object of this class type per user account.
-
-    """
-    def __init__(self, name):
-        self._id = name
-        self._remotes = []
-        self._config = {}
-        self._load_config()
-
+class User(db.Model):
+    __tablename__ = 'users'
+    id = db.Column('user_id',db.Integer, primary_key=True)
+    username = db.Column('username', db.String(20), unique=True, index=True)
+    password = db.Column('password', db.String(10))
+    
+    def __init__(self, username,password):
+        self.username = username
+        self.password = password
+         
+    def is_authenticated(self):
+        return True
+ 
+    def is_active(self):
+        return True
+ 
+    def is_anonymous(self):
+        return False
+ 
     def get_id(self):
-        """This class attrubute holds the user account's ID"""
```

Comment:
Publicly available functions need a docstring. Please add this back in... You may need to adjust what the docstring says since the `self._id` variable seems to hold different info now.

Reply:
I believe that function is still in there I just have two of the same functions so I removed the duplicate on.

Reply:
def get_id(self):
        """This class attrubute holds the user account's ID"""
        return str(self.id)

this is the function I have now for some reason github cut it out.